<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoichiometryDisplayWidget</class>
 <widget class="QWidget" name="StoichiometryDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="ElementTableView" name="dataView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="statusDescriptionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/add_icon</normaloff>:/add_icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/remove_icon</normaloff>:/remove_icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 1 5 1 5;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../icons/icons.qrc">:/status_unknown</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElementTableView</class>
   <extends>QTableView</extends>
   <header>GEMEditor.widgets.views</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>StoichiometryDisplayWidget</receiver>
   <slot>add_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_button</sender>
   <signal>clicked()</signal>
   <receiver>StoichiometryDisplayWidget</receiver>
   <slot>delete_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_item()</slot>
  <slot>delete_item()</slot>
 </slots>
</ui>
